tests/unit/lms/views/api/exceptions_test.py

```diff
@@ -118,3 +102,8 @@ def context():
 @pytest.fixture
 def views(context, pyramid_request):
     return APIExceptionViews(context, pyramid_request)
+
+
+@pytest.fixture(autouse=True)
+def report_exception(patch):
+    return patch("lms.views.api.exceptions.report_exception")
```

Comment:
`lms/views/api/exceptions.py` not imports `h_pyramid_sentry.report_exception()`. We don't want any tests to start accidentally making network requests to Sentry so we add this module-scoped autouse text fixture to ensure that `report_exception()` is always patched for all tests in this module. This is why patch fixtures should always be module-scoped and have `autouse`.